docs/development/potential-features.rst

```diff
@@ -0,0 +1,117 @@
+Potential Features
+==================
+
+These are potential feature ideas for Mathesar, but are not slated to be built yet. Each section represents a conceptual grouping of features. 
+
+Features slated to be built are not listed here, see the :doc:`Roadmap <roadmap>`.
+
+.. warning:: We are still finalizing the roadmap and it is expected to change significantly over the next few weeks. This notice will be removed when the roadmap is more stable.
+
+.. contents:: :local:
+
+Additional Imports
+------------------
+Users should be able to import data in the following additional formats:
+
+- SQL
+- JSON
+- XML
+- Google Sheets import (via API)
+- Excel file upload
+- Excel web import (via API)
+- Apple Numbers file upload
+- Collabora import
+- Airtable
+
+Additional Fields
+-----------------
+- File field (for images, attachments, etc.)
+- IP Address field
+- Formula field (use of spreadsheet like formulas)
```

Comment:
An intermediate step might be to give users the ability to make a generated column using SQL if they know how.